### PR TITLE
Refactor javadoc (Add, Delete, Find, List, View, Clear, Done, Rpreview, Exit) - Refactor rpreview method lengths

### PR DESCRIPTION
**Pull request by helloImHai**
- Shorten length of method in ResumePreviewCommand
- Add execute to shared commands and Hai's commands

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/Command.java [hunk 1]
```diff
@@ -13,8 +13,9 @@
      * Executes the command and returns the result message.
      *
      * @param model {@code Model} which the command should operate on.
-     * @return feedback message of the operation result for display
-     * @throws CommandException If an error occurs during command execution.
+     * @return      results from executing the command.
+     *
+     * @throws      CommandException If an error occurs during command execution.
```
Comment (chrisjwelly) [inline src/main/java/seedu/address/logic/commands/Command.java]: I think you missed out the newline in this one

File: src/main/java/seedu/address/logic/commands/ResumePreviewCommand.java [hunk 2]
```diff
@@ -99,10 +87,36 @@ private String getDataFromResume(Resume resume, Model model) {
...
+
+    private List<String> getStringListOfSkills(Resume resume, Model model) {
+        return resume.getSkillIds()
+                .stream()
+                .map(x -> model.hasSkillId(x) ? model.getSkillById(x) : null)
+                .filter(Objects::nonNull)
+                .map(Skill::toPreview)
+                .collect(Collectors.toList());
+    }
+
```
Comment (chrisjwelly) [inline src/main/java/seedu/address/logic/commands/ResumePreviewCommand.java:119]: Can you try running checkstyleMain locally? When I did this, I got scolded to write Javadocs for them :'(

File: src/main/java/seedu/address/model/item/exceptions/DuplicateItemException.java [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
- * identity).
+ * Signals that the operation will result in duplicate Items (Item are considered duplicates if {@Code isSame} function
+ * returns true).
  */
 public class DuplicateItemException extends RuntimeException {
     public DuplicateItemException() {
-        super("Operation would result in duplicate persons");
+        super("Operation would result in duplicate items");
     }
 }
```
Comment (chrisjwelly) [inline src/main/java/seedu/address/model/item/exceptions/DuplicateItemException.java:11]: Good catch!

**Review by chrisjwelly — COMMENTED** [review]
Several minor changes required here!

File: src/main/java/seedu/address/logic/commands/ResumePreviewCommand.java (hunk 2 above)
Comment (helloImHai) [reply src/main/java/seedu/address/logic/commands/ResumePreviewCommand.java:119]: My checkstyle runs fine, maybe it's cause the function is short?

File: src/main/java/seedu/address/model/item/exceptions/DuplicateItemException.java (hunk 3 above)
Comment (helloImHai) [reply src/main/java/seedu/address/model/item/exceptions/DuplicateItemException.java:11]: Thanks m8

File: src/main/java/seedu/address/logic/commands/Command.java (hunk 1 above)
Comment (helloImHai) [reply src/main/java/seedu/address/logic/commands/Command.java]: Great catch, I'll push the change now

File: src/main/java/seedu/address/logic/commands/ResumePreviewCommand.java (hunk 2 above)
Comment (chrisjwelly) [reply src/main/java/seedu/address/logic/commands/ResumePreviewCommand.java:119]: Hmm perhaps. All right then if checkstyleMain doesn't flag anything!

**Review by chrisjwelly — APPROVED** [review]
LGTM! :tada:

**Pull request merged**